config/application.travis.yml

```diff
@@ -0,0 +1,5 @@
+production:
+  production_url: ""
```

Comment:
If we create a development deployment then I think we can use deployment_url, deployment_user, etc... But I don't see a use case where deployment_url or deployment_user would be used. 

Should I just remove application.travis.yml or rename it to application.example.yml since we have no current use cases?